pyproject.toml

```diff
@@ -26,12 +26,11 @@ classifiers = [
 requires-python = ">=3.9"
 dynamic = ["version"]
 dependencies = [
-  "vllm>=0.5.2",
+  "vllm>=0.5.3.post1",
   "prometheus_client==0.20.0",
   "grpcio==1.62.2",
   "grpcio-health-checking==1.62.2",
   "grpcio-reflection==1.62.2",
-  "transformers==4.42.4",
```

Comment:
`transformers` here is a `vllm` dependency aswell. Pinning it here can cause dependency issues.

Reply:
The main issue this was causing is that vllm>=0.5.3 depends on `transformers >= 4.43.1`, while we had pinned `4.42.4`, this was causing `pip install .[tests]` to re-install vLLM 0.5.3 over `VLLM_VERSION_OVVERIDE=git+https://github.com/vllm-project/vllm@main` because of the conflicting versions

Reply:
nice!